### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by drewbolles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,11 +55,13 @@ Savvy Widget supports a number of Javascript callbacks that you can use for anal
...
-      // results (e.g. if you intend to use Savvy API to render the results natively).
-      // Otherwise, leave undefined.
-      experience: undefined,
+
+      // OPTIONAL:
+      // - Set the experience to 'connect' if you want to prevent Savvy SDK from displaying
+      //   results (e.g. if you intend to use Savvy API to render the results natively).
+      // - Set the experience to 'direct-login' if you want to skip the intermediary login interview questions
+      // - Otherwise, leave undefined.
+      experience: undefined | 'connect' | 'direct-login',
```
Comment (ddemetri) [inline README.md]: @drewbolles No way, Jose. 1) What is the client wants BOTH "connect" and "direct-login"? These are not mutually exclusive, so they should not be on the same field. 2) Let's have code examples that are copy-pastable. I.e. the allowed values should be in the JS comment and not in the code.

**Review by ddemetri — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (drewbolles) [reply README.md]: I added a new boolean field for this configuration property

File: README.md [hunk 2]
```diff
@@ -55,12 +55,15 @@ Savvy Widget supports a number of Javascript callbacks that you can use for anal
...
       // OPTIONAL: Your Trellis Client ID. Required if you intend to collect end-user PII.
       trellisClientId: API_CLIENT_ID,
-  
+
       // OPTIONAL: Set the experience to 'connect' if you want to prevent Savvy SDK from displaying
       // results (e.g. if you intend to use Savvy API to render the results natively).
       // Otherwise, leave undefined.
       experience: undefined,
 
+      // OPTIONAL: Set this value to true if you would like to skip the intermediary login interview questions
```
Comment (ddemetri) [inline README.md]:
```suggestion
      // OPTIONAL: Set true to skip qualification questions (e.g. "Do you remember your login?") prior to the credentials page.
```

File: README.md [hunk 3]
```diff
@@ -55,12 +55,15 @@ Savvy Widget supports a number of Javascript callbacks that you can use for anal
...
+      directLogin: false,
```
Comment (ddemetri) [inline README.md]:
```suggestion
      skipQualificationQuestions: false,
```

**Comment by drewbolles** [comment]
@ddemetri I merged in your suggestions, should be good to go

**Pull request merged**